app/remoteCam/video.py

```diff
@@ -73,11 +73,11 @@
     d = output[:-9]
     d = d.replace("-", " ").replace("T", " ").replace(":", " ")
 
-    starttime = datetime.strptime(d, "%Y %m %d %H %M %S %z")
+    starttime = datetime.strptime(d, "%Y %m %d %H %M %S")
```

Comment:
I see this is a "drive-by" bugfix which isn't related to the subject of the pull request.

Ideally this would be in a separate PR which relates to a separate bug report in a github issue.

Less ideally, but still better, is for it to be in a separate commit (perhaps with "drive-by bug fix" in the commit comment).   That git could explain what the bug fix is about (in the absence of a git issue about it).

To summarise, in an ideal world, each commit does a set of tightly-related things, and a PR is a set of commits related to a general topic.

That said I wouldn't waste time fixing that here, just something to bear in mind :)

Reply:
Thanks for pointing this out, that wasn't meant to change. I'll change it back now.

Reply:
Oh wait, I remember now. I put the %z in the wrong line, it got added to the correct line in master but it wasn't removed from the wrong one. This was fixed about a week ago when Dad was doing stuff so it wasn't pushed to git.